packages/components/input/src/textarea.tsx

```diff
@@ -122,13 +121,25 @@ const Textarea = forwardRef<"textarea", TextAreaProps>(
       />
     );
 
+    const end = useMemo(() => {
+      if (isClearable) {
+        return (
+          <button {...getClearButtonProps({className: "top-2"})}>
```

Comment:
why we need `top-2` here?

Reply:
because the clear button is absolutely positioned on the outermost div element and its original behavior is: 
![image](https://github.com/user-attachments/assets/0d5aedcc-1470-4d52-80fb-03d0ebf02973)
so i have to add **top-2** classname to position it at the top:
![image](https://github.com/user-attachments/assets/529214fa-d6aa-44f1-8f0d-b04c5af931f2)

Reply:
i think you may revise the structure based on the new design. It shouldn't require `top-2` at all in the new design.

Reply:
ok

Reply:
I don't understand why this clear button has a default top position

Reply:
sorry, could give me some tips, how to revise the structure?

Reply:
sorry, could u give me some tips, how to revise the structure?

Reply:
label and clear button will be on the same row. currently clear button is using absolute which is no longer required.

Reply:
thanks，i know how to do it.

Reply:
![image](https://github.com/user-attachments/assets/0d4befd2-cd4c-4f04-a81c-c76477519179)
![image](https://github.com/user-attachments/assets/79fd4d0b-59c0-461d-a0b6-f1fde6969f9b)